aea/cli/registry/main.py

```diff
@@ -0,0 +1,15 @@
+import click
```

Comment:
This `main.py` is not required. We already instantiate a cli here: https://github.com/fetchai/agents-aea/blob/develop/aea/cli/__main__.py